### ISSUE TITLE
Change teleport button

### ISSUE THREAD

**Issue by dmarcos** [aframevr/a-painter#214]
It is now mapped to the touchpad. After giving a lot of demos most the people trigger the teleport by mistake and it's very confusing.

**Comment by avrljk** [comment]
Potentially related:  every time I teleport it maximises the brush size.  Are beginners accidentally teleporting instead of resizing the brush?

If so, how about remapping brush resize from vertical swipe on the touch pad to horizontal swipe?  Horizontal is more intuitive because it matches the orientation of the brush size slider above the colour picker.

That way, very frequent teleporters like me won't have to reset their brush size every time they move around their work.  Beginners may be less likely to teleport too.

For Oculus Touch, teleport is on the B / Y buttons .  Brush size is up/down with either joystick.  I'd guess that could easily be changed to left/right so it also matches the brush slider on the tools palette.

Another aid to beginners might be to flash the teleport button on the controller model for a few seconds after a teleport (maybe temporarily label it like the other controls).  That behaviour could be turned off at the same time the controller labels are faded out.

**Comment by Utopiah** [comment]
Indeed it's a typical problem

**Comment by fernandojsg** [comment]
Yep I agree. In fact I was having a conversation yesterday with @dmarcos about this specific issue.
I like the proposal of @avrljk to change the slider from vertical to horizontal (red part), and maybe use just the top part (yellow) of the controller to detect the click, if you click in another place it won't do the teleporting.

![image](https://user-images.githubusercontent.com/782511/30699546-5188d1a0-9ee5-11e7-8149-7719d5e96763.png)

It could be a temporary fix until we'll manage how to remap the controls, for example having asymmetric controls.

**Comment by Utopiah** [comment]
Somehow people feel like they need to use their thumbs... Maybe they don't
have a proper grasp of the controller but really often I have to physically
prevent them from using it.

We could also work on a small introductory tutorial in addition to the
tooltip at the beginning, something similar to Google Blocks introduction.

**Comment by avrljk** [comment]
At our multi-user A-Painter demos, in the first 30 seconds, I get new A-Painters to draw their first couple of brushstrokes, side-step and poke their head through them.  Then it's a quick demo of how to teleport forward then turn around to see the back of their work

It really helps novices "get" the fact that they're in 3D, not just painting on a flat surface.  The usual reaction is "That's so cool!", first for the head poking then again for the teleport / turn.

After that they don't get confused by teleporting accidentally.

A tutorial sounds like a great idea.  I was also thinking of doing a video for people to watch on their phones while they're in the queue when we exhibit.
